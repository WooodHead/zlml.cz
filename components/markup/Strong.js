// @flow

type Props = {
  children: Object,
};

export default function Strong(props: Props) {
  return <strong {...props} />;
}
