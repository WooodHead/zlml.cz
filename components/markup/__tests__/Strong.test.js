// @flow

import React from 'react';
import renderer from 'react-test-renderer';

import Strong from '../Strong';

it('renders correctly', () => {
  expect(renderer.create(<Strong>TEST</Strong>).toJSON()).toMatchSnapshot();
});
