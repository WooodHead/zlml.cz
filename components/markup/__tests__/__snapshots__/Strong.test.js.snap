// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<strong>
  TEST
</strong>
`;

exports[`renders correctly 2`] = `
<strong
  style={
    Object {
      "color": "green",
    }
  }
>
  TEST
</strong>
`;
